packages/volto/src/components/manage/UniversalLink/UniversalLink.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { HashLink as Link } from 'react-router-hash-link';
+import { useSelector } from 'react-redux';
+import {
+  flattenToAppURL,
+  isInternalURL,
+  URLUtils,
+} from '@plone/volto/helpers/Url/Url';
+import config from '@plone/volto/registry';
+import cx from 'classnames';
+
+interface UniversalLinkProps {
+  href: string;
+  item?: {
+    '@id': string;
+    remoteUrl?: string;
+    getRemoteUrl?: string;
+    '@type'?: string;
+  };
+  openLinkInNewTab?: boolean;
+  download?: boolean;
+  children: React.ReactNode;
+  className?: string;
+  title?: string;
+  smooth?: boolean;
+  onClick?: (e: React.MouseEvent) => void;
+  onKeyDown?: (e: React.KeyboardEvent) => void;
+}
+
+export interface AppState {
+  content: {
+    data?: {
+      '@components'?: {
+        translations?: {
+          items?: { language: string; '@id': string }[];
+        };
+      };
+    };
+  };
+  navroot: {
+    data: {
+      navroot: {
+        id: string;
+      };
+    };
+  };
+  intl: {
+    locale: string;
+  };
+  userSession: {
+    token: string;
+  };
+}
+
+const UniversalLink = React.memo<UniversalLinkProps>(
+  ({
+    href,
+    item,
+    openLinkInNewTab,
+    download,
+    children,
+    className,
+    title,
+    smooth,
```

Comment:
`smooth` comes from the config always below. If you want it to be overridable, it's ok, but then enable it below.